string path
float64 pixelPerMeter

---
